app/services/hyrax/batch_ingest/batch_runner.rb

```diff
@@ -25,46 +26,47 @@ def initialize_batch
       end
 
       def read
+        raise ArgumentError, "Batch not initialized yet" unless batch.persisted?
         reader = config.reader.new(batch.source_location)
         batch.batch_items = reader.batch_items # batch item initialized (and now persisted)
-        if batch.submitter_email.present? && reader.submitter_email.present?
-          if batch.submitter_email != reader.submitter_email
-            notify_conflict(reader)
-          else
-            batch.submitter_email = reader.submitter_email
-          end
-        end
-        batch.status = :accepted
+        ensure_submitter_email(reader)
+        batch.status = 'accepted'
         batch.save! # batch accepted
       rescue ReaderError, ActiveRecord::ActiveRecordError => e
         notify_failed(e)
       end
 
       def enqueue
-        return if batch.batch_items
+        raise ArgumentError, "Batch not read yet" unless batch.status == 'accepted'
+        # notify_failed("No batch items found.") if batch.batch_items.blank?
         batch.batch_items.each do |item|
-          BatchItemProcessingJob.perform_later(item, config.source_validator, config.mapper)
-          batch_item.update(status: :enqueued) # batch item enqueued
+          BatchItemProcessingJob.perform_later(item)
+          item.update(status: 'enqueued') # batch item enqueued
         end
-        batch.update(status: :enqueued) # batch enqueued
+        batch.update(status: 'enqueued') # batch enqueued
         # TODO: Send email that batch has been enqueued
       rescue ActiveRecord::ActiveRecordError => e
         notify_failed(e)
       end
 
       private
 
+        def ensure_submitter_email(reader)
+          if reader.submitter_email.present?
+            if batch.submitter_email.present? && batch.submitter_email != reader.submitter_email
+              raise ReaderError, "Conflict: Different submitter emails found (#{batch.submitter_email} and #{reader.submitter_email})"
```

Comment:
@cjcolvar just to be clear, this implies that the submitter email in the Batch _must_ match the submitter email (if any) that the Batch was initiated with?